docs/sources/explore/index.md

```diff
@@ -155,6 +155,10 @@ Along with metrics, Explore allows you to investigate your logs with the followi
 - [InfluxDB](../datasources/influxdb)
 - [Elasticsearch](../datasources/elasticsearch)
 
+### Logs visualization
+
+Results of log queries are shown as histograms in graph and individual logs are displayed below. If datasource does not send histogram data for the requested time range, the logs model computes a timeseries based on the log row counts bucketed by an automcatically calculated time interval. The start of histogram is anchored by the first log row's timestamp from the result. The end of the timeseries is anchored to the timepicker's **To** range.
```

Comment:
> The start of histogram is anchored by the first log row's timestamp from the result.

This is only true if we compute the histogram from the logs data (so for Loki, maybe influx?). Not sure if that should be clear from the text but wasn't to me.

Reply:
```suggestion
Results of log queries are shown as histograms in graph and individual logs are displayed below. If datasource does not send histogram data for the requested time range, the logs model computes a time series based on the log row counts bucketed by an automatically calculated time interval and the start of histogram is then anchored by the first log row's timestamp from the result. The end of the time series is anchored to the timepicker's **To** range.
```

Reply:
@aocenas

Reply:
Sounds better :+1: